client/pages/org-edit-modal/modal.template.html.slim

```diff
@@ -4,19 +4,13 @@ form#org-edit-modal novalidate=true name='orgForm' ng-submit='orgForm.$valid &&
     h3.modal-title ng-if='editedOrg.id' Edit {{ organization | orgName }}
     h3.modal-title ng-if='!editedOrg.id' New Organization
   .modal-body
-    .form-group ng-class='{"has-error": orgForm.name.$dirty && orgForm.name.$invalid}'
-      label for='orgName' Name
-      input#orgName.form-control type='string' name='name' placeholder='Name (used in URLs)' required=true maxlength=50 ng-pattern='/^[a-z0-9]+(?:-[a-z0-9]+)*$/i' unique-org-name=true autofocus=true ng-model='editedOrg.name' ng-model-options='{updateOn: "default blur", debounce: {default: 500, blur: 0}}'
+    .form-group ng-class='{"has-error": orgForm.displayName.$dirty && orgForm.displayName.$invalid}'
+      label for='orgDisplayName' Name
+      input#orgDisplayName.form-control type='string' name='displayName' maxlength=50 ng-model='editedOrg.displayName' unique-org-name=true
```

Comment:
You should not have removed `ng-model-options='{updateOn: "default blur", debounce: {default: 500, blur: 0}}'`.